tests/medium/plugins/test_jinja_generator.py

```diff
@@ -0,0 +1,36 @@
+import os
+import sys
+from mock import patch
+from splunk_eventgen.__main__ import parse_args
+from splunk_eventgen.eventgen_core import EventGenerator
+
+FILE_DIR = os.path.dirname(os.path.abspath(__file__))
+OUTPUT_FILE = 'test_jinja_generator_file_output.result'
+
+
+class TestJinjaGenerator(object):
+
+    def test_jinja_generator_to_file(self):
+        configfile = "tests/sample_eventgen_conf/jinja/eventgen.conf.jinja_basic"
+        testargs = ["eventgen", "generate", configfile]
+        file_output_path = os.path.abspath(os.path.join(FILE_DIR, '..', '..', '..', OUTPUT_FILE))
+        # clear the result file
+        open(file_output_path, 'w').close()
```

Comment:
if it is a file, os.remove will be more readable :)

Reply:
changed.